tests/testAfterburners.py

```diff
@@ -0,0 +1,181 @@
+import numpy as np
+
+import unittest
+import lsst.utils.tests as utilsTests
+
+import lsst.meas.base.afterburner as AB
+import lsst.afw.table as afwTable
+from lsst.meas.base.pluginsBase import BasePlugin
+from lsst.meas.base.pluginRegistry import register
+from lsst.meas.base.baseLib import MeasurementError
+
+@register("afterburnerFail")
+class FailAb(AB.AfterburnerPlugin):
+    '''
+    Afterburner plugin which is guaranteed to fail, testing the failure framework
+    '''
+    @classmethod
+    def getExecutionOrder(cls):
+        return cls.DEFAULT_AFTERBURNER
+
+    def __init__(self, config, name, schema, metadata):
+        AB.AfterburnerPlugin.__init__(self, config, name, schema, metadata)
+        self.failKey = schema.addField(name+"_fail", type="Flag", doc="Failure test")
+
+    def burn(self, measRecord):
+        raise MeasurementError("Supposed to fail", 0)
```

Comment:
A better way to write the zero might be `FlagHandler.FAILURE`, if that's available in Python (I don't recall whether it is).